.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'BUG: (Please describe your issue)'
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (if issue was encountered on a desktop):**
+ - OS with version: [e.g., macOS Sonoma 14.5]
+ - Browser [e.g., chrome, safari]
+
+**Mobile (if issue was encountered on a mobile device):**
+ - Device: [e.g., iPhone 15 Pro]
+ - OS with version: [e.g., iOS 17.5.1]
+ - Browser [e.g., stock browser, safari]
+
+**System/Environment (if issue was encountered while making a contribution to this repository):**
+- System CPU: [e.g., (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz]
+- System memory: [e.g., 16 GB]
+- Shell: [e.g., bash]
```

Comment:
```suggestion
```

this is awesome @BradonZhang i think we can remove the system CPU, memory and shell as i don't think that will impact the website build.